lib/include.js

```diff
@@ -47,6 +49,8 @@ function normalizeInclude(include) {
       newInclude = newInclude.concat(subIncludes);
     }
     return newInclude;
+  } else if (DISALLOWED_TYPES.includes(typeof include)) {
+    return [];
```

Comment:
Please add a debug statement to let the user know that we are discarding invalid `include` value.

```ts
debug('Ignoring invalid "include" value of type %s:', typeof include, include);
```